src/apiCalls.tsx

```diff
@@ -110,39 +110,50 @@ export function getSelectedCustomerOrders(id: number) {
 
 export function postNewCustomer(newCustomer: any) {
   return fetch(`${url}/customers/`, {
-    method: "POST",
+    method: 'POST',
     headers: {
-      "Content-Type": "application/json",
+      'Content-Type': 'application/json',
     },
     body: JSON.stringify(newCustomer),
   })
-    .then((response) => {
+    .then(response => {
       if (!response.ok) {
         throw new Error(`${response.status} ${response.statusText}`);
       }
       return response;
     })
-    .then((response) => response.json());
+    .then(response => response.json());
 }
 export function postNewVendor(newVendor: any) {
   return fetch(`${url}/vendors/`, {
-    method: "POST",
+    method: 'POST',
     headers: {
-      "Content-Type": "application/json",
+      'Content-Type': 'application/json',
     },
     body: JSON.stringify(newVendor),
   })
-    .then((response) => {
+    .then(response => {
       if (!response.ok) {
         throw new Error(`${response.status} ${response.statusText}`);
       }
       return response;
     })
-    .then((response) => response.json());
+    .then(response => response.json());
 }
 
 export function getSelectedVendorOrders(id: number) {
-  return fetch(`${url}/vendors/${id}/preorders/`).then((response) => {
+  return fetch(`${url}/vendors/${id}/preorders/`).then(response => {
+    if (!response.ok) {
+      throw new Error(`${response.status} ${response.statusText}`);
+    }
+    return response.json();
+  });
+}
+
+export function getMarkets() {
+  return fetch(
+    `https://quiet-depths-54407-77a00505f51e.herokuapp.com/markets/location/80206/10`
```

Comment:
So great to have this working with the endpoints the Back End gave us and with the radius too!  Next we'll want to interpolate the zip and radius so we can get the data for the user inputs of zip and radius